pulsar-sql/presto-pulsar/src/main/java/org/apache/pulsar/sql/presto/PulsarColumnMetadata.java

```diff
@@ -187,7 +188,7 @@ public boolean equals(Object o) {
 
             DecoderExtraInfo that = (DecoderExtraInfo) o;
 
-            if (mapping != that.mapping) {
+            if (!StringUtils.equals(mapping, that.mapping)) {
```

Comment:
no need to use a third party library for this.
we can use `Object.equals(mapping, that.mapping)`

Reply:
I also changed other comparisons to use `Objects.equals`, PTAL